compiler/resolution/wrappers.cpp

```diff
@@ -2232,7 +2232,13 @@ PromotionInfo::PromotionInfo(FnSymbol* fn,
       promotedType = ts;
     } else {
       bool actualProvidedForFormal = false;
-      for_vector(ArgSymbol, arg, actualFormals) {
+      // Can't use for_vector here as it will not iterate if the vector starts
+      // with a NULL entry
+      for (long unsigned int k = 0; k < actualFormals.size(); k++) {
+        ArgSymbol* arg = actualFormals[k];
+        if (arg == NULL)
+          continue;
```

Comment:
AFAIK this condition is not necessary because `formal` will not be `NULL` and arg is compared to formal on the next line. But if you do keep it you should probably write `nullptr` per the other pending PR.

Reply:
But that's a good point about the symbol being compared to `formal`, thanks!